libs/data-service-generator/src/admin/create-admin.ts

```diff
@@ -122,12 +127,21 @@ async function createAdminModulesInternal(): Promise<Module[]> {
   );
 
   logger.info("Formatting code...");
-  const formattedModules = createdModules.map((module) => ({
-    ...module,
-    code: formatCode(module.code),
-  }));
+  const formattedModules = createdModules.map((module) => {
+    if (module.path.includes(".gitignore")) {
+      return {
+        ...module,
+        code: module.code,
+      };
+    }
```

Comment:
Is that a must? Is there any other way to do that? .e.g can we tell the formatter to ignore unknown files?